src/main/interface/cli.c

```diff
@@ -2560,13 +2561,14 @@ static void cliBeeper(char *cmdline)
 }
 #endif
 
-#ifdef USE_RX_FRSKY_SPI
-void cliFrSkyBind(char *cmdline){
+#ifdef USE_RX_BIND
```

Comment:
Please add a second conditional `USE_CC2500_RX_BIND` here guarding the respective `case` statements, so that other RX bind methods can be added.

Reply:
fixed